packages/playwright/tests/axe-playwright.spec.ts

```diff
@@ -603,6 +639,21 @@ describe('@axe-core/playwright', () => {
     });
   });
 
+  describe('errorUrl', () => {
+    it('returns correct errorUrl', () => {
+      const errorUrl = (new AxeBuilder({ page }) as any).errorUrl;
+      assert.equal(
+        errorUrl,
+        'https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md'
+      );
+    });
+    it('return modified errorUrl', () => {
+      const builder = new AxeBuilder({ page }) as any;
+      builder.errorUrl = 'https://deque.biz';
+      assert.equal(builder.errorUrl, 'https://deque.biz');
+    });
```

Comment:
This isn't a useful test (checking if the value you just set is equal to the value you set)